riot/tft/constants.go

```diff
@@ -0,0 +1,65 @@
+package tft
+
+const (
+	endpointBase          = "/tft"
+	endpointSpectatorBase = "/lol/spectator" + endpointBase
```

Comment:
is this intended? This uses the `/lol` base path.

Reply:
Yes, actually, it uses /lol/ as base path for the spectator. https://developer.riotgames.com/apis#spectator-tft-v5